employees/tests/test_unit_report_viewset.py

```diff
@@ -481,7 +481,7 @@ def _assert_response_contain_report(self, response, reports):
             for date in dates:
                 fields_to_check.append(
                     datetime.datetime.strftime(
-                        datetime.datetime.fromtimestamp(int(getattr(report, date).timestamp())), "%B %d, %Y, %-I:%M"
+                        datetime.datetime.fromtimestamp(int(getattr(report, date).timestamp())), "%B %-d, %Y, %-I:%M"
```

Comment:
Make it constant.

Reply:
I did not see it before I merge it. Next time I will fix it :)